src/main/java/ru/ayaz/ChatRoom.java

```diff
@@ -0,0 +1,76 @@
+package ru.ayaz;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.BlockingQueue;
+
+public class ChatRoom implements Runnable {
+
+    private Map<String, User> userMap;
+    private Map<String, UserSocketHandler> userSocketHandlerMap;
+    private BlockingQueue<UserMessage> messageQueue;
+
+    ChatRoom(BlockingQueue<UserMessage> queue) {
+        this.userMap = new HashMap<>();
+        this.userSocketHandlerMap = new HashMap<>();
+        this.messageQueue = queue;
+    }
+
+    void enqueueMessage(UserMessage message) {
+        try {
+            messageQueue.put(message);
+        } catch (InterruptedException e) {
+            e.printStackTrace();
+        }
+    }
+
+    boolean queueContainsMessage(UserMessage message) {
```

Comment:
&#x1F34E; **Issue**
I would rename this to something more appropriate. Its not quite clear what it means for a room to have a queue and that queue to contain a message.
For how long does it contain it ? Can I enqueue several messages and expect them to be there ? This might be solved with a more user-friendly interface for the Room object.
Also, looks like this method is used just for the testing purposes. I don't think it's a very good approach to create helper-methods just to test something. It tells us that you are trying to test a part of some use case of your class and not the use case as a whole. I would go with the approach of testing based on necessary use cases.